nncf/onnx/graph/node_utils.py

```diff
@@ -53,3 +56,21 @@ def get_bias_value(node_with_bias : NNCFNode, model: onnx.ModelProto) -> np.ndar
     if metatype == ONNXIdentityMetatype:
         return onnx_graph.get_initializers_value(node.input[0])
     raise RuntimeError('Could not find the bias value of the node')
+
+
+def get_nncf_input_node_next_onnx_nodes(nncf_graph: NNCFGraph) -> Dict[str, Tuple[str, int]]:
+    """
+    Returns mapping between NNNCFGraph input nodes and following by ONNX nodes with corresponding input port ids.
+
+    :param nncf_graph: instance of NNCFGraph
+    :return: A mapping of NNNCF input node names and a tuple with the consumed node names and their input port ids.
+    """
+    nncf_input_node_next_onnx_nodes = {}
+    for input_node in nncf_graph.get_input_nodes():
+        nncf_input_node_next_onnx_nodes[input_node.node_name] = []
+        for next_node in nncf_graph.get_next_nodes(input_node):
+            for edge in nncf_graph.get_input_edges(next_node):
+                if edge.from_node == input_node:
+                    nncf_input_node_next_onnx_nodes[input_node.node_name].append((next_node.node_name,
```

Comment:
Why just not store the correct edge name in the mapping? Instead of packaging the `port_id` and using it for further name calculation.

Reply:
Yeah, this is probably better idea

Reply:
Discussed